src/components/Layout/drawer.js

```diff
@@ -43,7 +43,7 @@ class SideComponent extends Component {
           </ListItem>
         </Link>
         <Divider />
-        <ListSubheader>Categories</ListSubheader>
+        <ListSubheader>All</ListSubheader>

```

Comment:
Instead of a ListSubheader, it should look like the other categories. For now it can just link to the home page. More like this 
```
<Link
   to={`/`}
   className={classes.link}
 >
  <ListItem button>
    <ListItemText secondary='All' />
  </ListItem>
 </Link>
```

Reply:
alright i get what you mean